setup.py

```diff
@@ -1,60 +1,60 @@
-"""
-A setuptools based setup module.
```

Comment:
Have the line-endings of all files changed? I would really prefer it if you could re-do the PR without these invasive changes, as diffing is quite hard, and revision history is lost.